apprise/plugins/NotifyNtfy.py

```diff
@@ -0,0 +1,264 @@
+# MIT License
+
+# Copyright (c) 2022 Joey Espinosa <@particledecay>
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+import requests
+import six
+from json import loads
+
+from .NotifyBase import NotifyBase
+from ..common import NotifyType
+from ..AppriseLocals import gettext_lazy as _
+
+
+# Priorities
+class NtfyPriority(object):
+    URGENT = 'urgent'
+    HIGH = 'high'
+    DEFAULT = 'default'
+    LOW = 'low'
+    MIN = 'min'
+
+    @classmethod
+    def get_value(cls, priority):
+        if priority == NtfyPriority.DEFAULT:  # most common case
+            return 3
+        elif priority == NtfyPriority.URGENT:
+            return 5
+        elif priority == NtfyPriority.HIGH:
+            return 4
+        elif priority == NtfyPriority.LOW:
+            return 2
+        elif priority == NtfyPriority.MIN:
+            return 1
+
+
+
+NTFY_PRIORITIES = (
+    NtfyPriority.URGENT,
+    NtfyPriority.HIGH,
+    NtfyPriority.DEFAULT,
+    NtfyPriority.LOW,
+    NtfyPriority.MIN,
+)
+
+
+class NotifyNtfy(NotifyBase):
+    """
+    A wrapper for Ntfy Notifications
+    """
+
+    # The default descriptive name associated with the Notification
+    service_name = 'Ntfy'
```

Comment:
So technically it's "ntfy" not "Ntfy". I am not sure what the guidelines here are, though. Android for instance wants the name of the app to start with a capital letter, but it's officially "ntfy".